addrecipe.html

```diff
@@ -17,24 +17,38 @@
 </head>
 
 <body>
-    <nav class="uk-navbar-container" uk-navbar>
-        <div class="uk-navbar-left">
-            <ul class="uk-navbar-nav">
-                <li><a href="index.html">Home</a></li>
-                <li class="uk-active"><a href="addrecipe.html">Add recipe</a></li>
-            </ul>
-        </div>
-    </nav>
+  <nav class="uk-navbar-container" uk-navbar>
+    <div class="uk-navbar-left">
+      <ul class="uk-navbar-nav">
+        <li><a href="index.html">Home</a></li>
+        <li class="uk-active"><a href="addrecipe.html">Add recipe</a></li>
+      </ul>
+    </div>
+  </nav>
+
 
+  <!-- <div class="uk-margin">
+    <form class="uk-search uk-search-default">
+      <label for="searching" class="size-10">WHAT ARE YOU LOOKING FOR?</label>
+      <a href="" class="uk-search-icon-flip" uk-search-icon></a>
+      <input class="uk-search-input" type="search" name="searching" placeholder="Search...">
+    </form>
+  </div> -->
+
+  <form class="uk-search uk-search-default margin-20">
     <div class="uk-margin">
-        <form class="uk-search uk-search-default">
-            <a href="" class="uk-search-icon-flip" uk-search-icon></a>
-            <input class="uk-search-input" type="search" placeholder="Search...">
-        </form>
+      <label for="searching" class="size-13">WHAT ARE YOU LOOKING FOR?</label>
```

Comment:
What does the `for` attribute do? What does it refer to?

Reply:
The first element in the document with an id matching the value of the for attribute is the labeled control for this label element, if it is a labelable element. But it's not necessary here.